### PR TITLE
One port

### PR DESCRIPTION
**Pull request by natew**

### CODE REVIEW

File: packages/vxrn/src/create.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import viteInspectPlugin from 'vite-plugin-inspect'
 import { readFile } from 'fs/promises'
 import { dirname, join, relative, resolve } from 'node:path'
+import getPort from 'get-port'
```
Comment (Aslemammad) [inline packages/vxrn/src/create.ts:4]: This can be good too. https://unjs.io/packages/get-port-please

File: packages/vxrn/src/create.ts [hunk 2]
```diff
@@ -49,7 +50,9 @@ const extensions = [
 ]
 
 export const create = async (options: StartOptions) => {
-  const { host = '127.0.0.1', root, nativePort = 8081, webPort } = options
+  const { host = '127.0.0.1', root, port = 8081 } = options
```
Comment (Aslemammad) [inline packages/vxrn/src/create.ts:53]: Why don't we assign a random port to this port too?

File: packages/vxrn/src/create.ts [hunk 3]
```diff
@@ -49,7 +50,9 @@ const extensions = [
...
+  const nativePort = await getPort()
```
Comment (Aslemammad) [inline packages/vxrn/src/create.ts:54]: I don't see it being used anywhere, so maybe we can remove it?

File: packages/vxrn/src/create.ts (hunk 2 above)
Comment (natew) [reply packages/vxrn/src/create.ts:61]: We can. But ideally default to 8081 and let user override like this

File: packages/vxrn/src/create.ts (hunk 3 above)
Comment (natew) [reply packages/vxrn/src/create.ts]: Yea lots of old test code feel free to keep going

File: packages/vxrn/src/create.ts (hunk 2 above)
Comment (szymonrybczak) [reply packages/vxrn/src/create.ts:61]: Let's keep 8081 as default - to be consistent with Metro (and also React Native apps looks by default for this port).

**Pull request merged**